assets/app/views/settings.html

```diff
@@ -1,119 +1,236 @@
 <div class="content">
   <project-page>
-    <div>
-      <div class="page-header page-header-bleed-right">
-        <delete-link
-          class="pull-right fa-lg"
-          resource-type="project"
-          resource-name="{{project.metadata.name}}"
-          project-name="{{project.metadata.name}}"
-          display-name="{{(project | displayName)}}"
-          alerts="alerts"
-          button-only>
-        </delete-link>
-        <h1>Project Settings</h1>
-      </div>
+    <div class="breadcrumb"></div>
+    <alerts alerts="alerts"></alerts>
+    <div class="tile__">
+      <h1>
+        Project Settings
+        <div class="pull-right dropdown">
+          <a href=""
+            class="dropdown-toggle resource-actions-dropdown"
+            data-toggle="dropdown">
+            <i class="fa fa-ellipsis-v"></i>
+            <span class="sr-only">Actions</span>
+          </a>
+          <ul class="dropdown-menu actions action-button">
+            <li>
+              <a href=""
+                role="button"
+                ng-click="toggle()"
+                ><i class="fa fa-pencil fa-fw" aria-hidden="true"></i>Edit</a>
```

Comment:
I think the difference in margins is because there is no space before text.

Does it look better as `<i class="fa fa-pencil fa-fw" aria-hidden="true"></i> Edit`?